autodns-cli

```diff
@@ -7,13 +7,16 @@ declare -r prog=$(basename "$0")
 declare -r endpoint=https://api.autodns.com/v1
 declare -r ttl=120
 
-if [ -f './.autodns-cli.rc' ]; then
+if [ -f '~/.autodns-cli.rc' ]; then
```

Comment:
Can you make it possibly load either? I am not sure what the best load order would be, maybe home first, then local second?